configs/base.tsconfig.json

```diff
@@ -14,13 +14,13 @@
         "resolveJsonModule": true,
         "module": "commonjs",
         "moduleResolution": "node",
-        "target": "es5",
+        "target": "ES2017",
         "jsx": "react",
         "lib": [
-            "es6",
+            "ES2017",
             "dom"
         ],
         "sourceMap": true,
         "composite": true
     }
-}
+}
```

Comment:
I'll amend the fix commit to restore the newline :+1:

Reply:
I do not think I even opened that file. Can such an edit happen on its own?

Reply:
The fix commit is now on the bottom, so maybe better that I amend my commit?

Reply:
I did in the fix (to update to es2017) :)
- a475368f24dfd05f298ce7e1b1a418f1dfee71ea